charts/opendj/templates/backup/cronjob.yaml

```diff
@@ -30,6 +30,9 @@ spec:
               kubectl cp \
```

Comment:
Asking for my own education -- what populates the `/opt/opendj/data/bak` directory?

Reply:
I've configured OpenDJ to run a nightly backup to that folder. I'll do the same for all the other envs. The tasks backend is included in the backup itself, so when an openDJ is restored from this backup it will resume backing itself up.

```[14:02:29] gmalkov:~ $ kubectl exec \
  --namespace "terra-${TERRA_ENV}" \
  --tty \
  --stdin \
  opendj-statefulset-0 \
  -c opendj -- /bin/bash -c "/opt/opendj/bin/manage-tasks \
    --hostname localhost  \
    --port 4444  \
    --bindDN \"cn=Directory Manager\" \
    --bindPassword ${OPENDJ_PASSWORD} \
    --info 03052021 \
    --trustAll"


  Task Details
  ----------------------------------------
  ID                        03052021
  Type                      Backup
  Status                    Recurring
  Scheduled Start Time      00 02 * * *
  Dependencies              None
  Failed Dependency Action  None
  Email Upon Completion     None Specified
  Email Upon Error          None Specified

  Backup Options
  ----------------------------------------------
  ds-task-backup-compress   true
  ds-task-backup-all        true
  ds-backup-id              03052021
  ds-backup-directory-path  /opt/opendj/data/bak
  objectclass               top
                            ds-task
                            ds-recurring-task
                            ds-task-backup
```

Reply:
I will include all of this info+commands in a write-up in the devops gitbook portal